uber/decorators.py

```diff
@@ -704,6 +704,24 @@ def food_price(self):
     """
 
 
+class receipt_item(property):
```

Comment:
I need to update this comment and specify that you actually return a third value, the receipt item type.